ddl-10-11/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(main)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(OpenCV_DIR /home/jin/下载/opencv-3.4.11/build)
+find_package(OpenCV REQUIRED)
+
+include_directories(/home/jin/下载/mindvision/include)
```

Comment:
do you install the SDK of mindvision on your ubuntu? Please tell me why you add this sentence.